proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -156,84 +156,84 @@ message PayloadOrganizationMemberRemoved {
   buf.alpha.registry.v1alpha1.OrganizationRole member_role = 3;
 }
 
-message PayloadRepositoryCreated {
-  // owner_id is the id of the owner of the repository.
+message PayloadModuleCreated {
```

Comment:
The motivating [slack thread]() only mentioned the enums, but I assumed that we wanted to update the message names as well -- is this assumption valid? should I drop this change?